tests/main/interfaces-network-setup-control/task.yaml

```diff
@@ -4,33 +4,31 @@ details: |
     The network setup control interface allows to access the different netplan
     configuration files.
 
-    The test uses the test-snapd-check-fs-access snap to checks files and dirs
-    are accessible through the interface.
-
 systems: [-ubuntu-core-*]
 
 prepare: |
-    echo "Given the test-snapd-check-fs-access snap is installed"
-    cp "$TESTSLIB"/snaps/test-snapd-check-fs-access/meta/snap.yaml "$TESTSLIB"/snaps/test-snapd-check-fs-access/meta/snap.yaml.orig
-    sed 's/\[\]/\[network-setup-control\]/g' -i "$TESTSLIB"/snaps/test-snapd-check-fs-access/meta/snap.yaml
-    snap try "$TESTSLIB"/snaps/test-snapd-check-fs-access
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local test-snapd-sh
 
 restore: |
-    cp "$TESTSLIB"/snaps/test-snapd-check-fs-access/meta/snap.yaml.orig "$TESTSLIB"/snaps/test-snapd-check-fs-access/meta/snap.yaml
+    rm -f call.error
+    rm -f /etc/netplan/test001 /etc/netplan/test002
```

Comment:
This line and the following line look exactly identical. Is this intentional or a copy/paste error?